README.md

```diff
@@ -43,7 +43,7 @@ We have several additional constraints that must be satisfied in order for a
 route map to be feasible.
 
  1. Every leg must connect to a hub.  
- 2. Only one hub per route.  
+ 2. Passengers only connect at hub airports.  
```

Comment:
Maybe it would be more clear to specify both these statements.
ie. "2. Passengers only connect at hub airports (meaning one hub per route). I'll defer to what you think is most clear here.

Reply:
They may have two hubs in a route though - that's why I'm updating this.  They can have as many connections as they want - provided they are only at hub airports.

Reply:
Understood 👌